src/hyperfine/benchmark.rs

```diff
@@ -132,10 +137,10 @@ pub fn mean_shell_spawning_time(
                 times_user.push(r.time_user);
                 times_system.push(r.time_system);
             }
-        }
-        progress_bar.inc(1);
+        }  
+        // if let Some(bar) = progress_bar { bar.inc(1); }
```

Comment:
what's going on here?

Reply:
oh, that was just a bug due to my struggling with borrowing. fixed in the update